docs/MIGRATION.md

```diff
@@ -0,0 +1,35 @@
+# Migration from AICSImageIO to BioIO
+
+## How different is the API?
+
+Very little about the API has changed between `aicsimageio` version 4.0.0+ and `bioio`, the API is largely the same with the biggest differences being:
+* You use a different Python package (`bioio` instead of `aicsimageio`)
+    * **Important** - See the [Reader Installation Instructions](https://github.com/bioio-devs/bioio/main/README.md#reader-installation) for which additional packages (`bioio` plug-ins) you'll need to support the files you want to interact with
+* The main class exported was renamed from `AICSImage` to `BioImage`
```

Comment:
I would add a bullet above this one:
`* Reader plugins are separate packages that are installed separately.  OmeTiffReeader and OmeZarrReader are preinstalled with the default distribution of bioio. `
(or something to that effect)

Reply:
Are we decided on including `bioio-ome-zarr` and `bioio-ome-tiff` in the `bioio` dist? I'm just wondering if that has the possibility to cause conflicts with other readers down the line, if you don't think that'll be a concern then I'm down

Reply:
Ok - Cancel that sentence until we can be definitive about that.    I just thought it would be convenient but you're probably right. Users should be able to decide.  This just brings up the hard problem of reader precedence, where date of install might not always be what the user needs.  Maybe we could make a "batteries included" option that preinstalls those but the main package doesn't.

Reply:
I'm not totally sure what is best but I'm amenable to either way.

Reply:
I could add a `default` extra installation option that installs `bioio-ome-zarr` and `bioio-ome-tiff`. Like so `pip install bioio[default]` 🤷

Reply:
I think the extra installation option is fine, but it's almost a wash compared to the effort of explicitly installing two extra readers.  We should have a simple example pip install command line in the documentation that shows people how to get the basic ome-tiff and ome-zarr readers.  (either by separate pip install or by bioio[default]